minutes/2020-03-17.md

```diff
@@ -0,0 +1,51 @@
+# Jakarta EE Platform Call
+
+Date: 2020-03-17 (unmeeting)
+
+**Note:**  This was an informal “unmeeting” this week. Ivar and Kevin attempted to cancel the meeting last minute due the lack of an agenda. But, several people showed up anyway and had a fruitful discussion. Bill Shannon took notes (thank you, Bill!).
+
+Present:
+
+- Bill Shannon (Oracle)
+- Emily Jiang (IBM)
+- David Blevins (Tomitribe)
+- Vano Beridze (???)
+- Others were also on the call, but no attendance was recorded or requested...
+
+
+## Agenda
+
+- None
+
+### Minutes (informal)
+
+- Despite the meeting being canceled, some people showed up anyway and we have some useful conversations.  Don't consider these minutes, they're just my recollections of what was talked about, and these are certainly not official minutes.  Corrections are encouraged.
+
+- Emily Jiang started by asking about the TCKs and lamenting how there was no documentation about the structure of the TCKs and thus it was difficult to determine how to extract pieces of the TCK repo for one particular API in order to create a standalone TCK for that API.
+  - I lamented that we had internal documentation on some of this that we've never found time to release.  Perhaps some of it would be useful, perhaps not, but I'll go back and see if there's any hope of springing it loose of Oracle.
+  - I reminded Emily to talk to Andy Guibert, who has figured out some parts of this.
+
+
+- We talked to David Blevins about EJB and what his plans are for EJB.  David was very clear that, given his resource and product requirements, he was not going to be able to have anyone help on the EJB implementation in GlassFish.  So, it's going to be up to the GlassFish project team to do this work.
+
+- We talked about the EJB interop tests.  David believes there are two kinds of EJB interoperability tests - tests that use only the EJB APIs to test interoperating with remote EJBs, presumably from the same vendor, and tests that use the CORBA APIs to test EJB interoperability using the IIOP, CSIv2, etc. protocols.  These later tests should be removed from the TCK.
+  - Note, however, that the only remote EJB protocol supported by GlassFish is the CORBA protocol, so these tests that would be removed from the platform TCK would be very useful as product tests for GlassFIsh, and we should be careful not to lose them or lose that ability to use them for GlassFish.  They would also be useful for any other products that currently provide this level of interoperability and are NOT planning to remove it for Jakarta EE 9 even though it is no longer part of the spec (not even optional).
```

Comment:
tests for GlassFIsh -> tests for GlassFish

Reply:
lose -> loose

Reply:
Good eyes!  Thanks.